docs/setup.md

```diff
@@ -21,24 +21,19 @@ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/
 
 ### 3. Install python, node.js, and npm
 
-Python may be required for working with some projects.  Installing node.js will provide you with the `node` and `npm` commands.
+Python may be required for working with some projects, though OSX has it built in. Installing node.js will provide you with the `node` and `npm` commands.
 
 ```
-brew install python nodejs;
-sudo easy_install pip;
-pip install pyobjc;
-npm install -g node-gyp;
+brew install nvm;
```

Comment:
@kangas Good find, I have updated the PR accordingly and also added a similar change for http://brew.sh/ immediately above, thanks.